Sources/Helpers/OAAppSettings.h

```diff
@@ -211,6 +211,21 @@ typedef NS_ENUM(NSInteger, EOAMapMarkersMode)
 
 @end
 
+@interface OAProfileStringList : OAProfileSetting
+
++ (instancetype) withKey:(NSString *)key defValue:(NSMutableArray *)defValue;
```

Comment:
NSMutableArray > NSArray<NSString *>
Since this is StringList all arrays params must be typed, i.e. <NSString *>